README.md

```diff
@@ -1 +1,17 @@
-# sclorg-testing-farm
\ No newline at end of file
+# FMF testing plans for usage in the Testing Farm service
+
+This repository contains testing plans, which are used by the [sclorg](https://github.com/sclorg/) testing GitHub Actions running on Testing Farm infrastructure.
+
+Available testing plans:
+
+| testing plan            | description               |
+|  --------------         |------------               |
+| `betka-bot.fmf`         | Runs betka tests on Fedora|
+| `betka-fedmsg.fmf`      | Runs betka-fedmsg tests on Fedora      |
+| `c9s.fmf`               | Runs docker tests on CentOS Stream 9   |
+| `fedora.fmf`            | not sure, **TODO**        |
```

Comment:
This will be used in the future for testing RHSCL containers on the Fedora host.